tests/mbed/host/CMakeLists.txt

```diff
@@ -7,7 +7,9 @@ oeedl_file(../mbed.edl host gen)
 add_executable(libmbedtest_host host.c ocalls.c ${gen})
 
 target_compile_definitions(libmbedtest_host PRIVATE PROJECT_DIR="${CMAKE_SOURCE_DIR}/")
-target_compile_options(libmbedtest_host PRIVATE -Wno-error)
+if (NOT WIN32)
+    target_compile_options(libmbedtest_host PRIVATE -Wno-error)
+endif()
 
-target_include_directories(libmbedtest_host PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
+target_include_directories(libmbedtest_host PRIVATE ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/..)
```

Comment:
`${CMAKE_CURRENT_SOURCE_DIR}` is redundant from here, it can just be `..`.

Reply:
ok, will incorporate.